File: credit/credit.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //TODO prompt for input
    long card;
    do
    {
        card = get_long("Number: ");
    }
    while (card < 0);

    //TODO calculate checksum
    //

    //TODO check for card length and starting digits
}

File: sentimental-credit/credit.py
import cs50
# TODO

def main():
    # Get user card input
    while True:
        try:
            card = get_int("Number: ")
            if card > 0:
                break
            else:
                print("Invalid number.")
        except ValueError:
            print("Please enter numbers only.")

    # checksum
    # print card type

def checksum(card):


if __name__ == "__main__":
    main()
